.travis.yml

```diff
@@ -27,11 +27,17 @@ before_script: # configure a headless display to test plot generation
     - "sh -e /etc/init.d/xvfb start"
     - sleep 3 # give xvfb some time to start
 
-script:
-    - flake8 parcels
-    - flake8 tests
-    - flake8 examples
-    - python parcels/scripts/pull_data.py
-    - py.test -v -s tests/
-    - py.test -v -s --nbval-lax examples/
-    - make linkcheck -C docs
+jobs:
+    include:
+        - script:
```

Comment:
Why separate these out into different scripts? The flake8 and linkcheck take no time compared to the unit tests in examples and parcels. But every script triggers a new conda-build, which seems a waste of resources? Unless there's a very good reason, I would keep just one script

Reply:
I did this, because it made it easier to locate the failing test. But I do see the point of keeping things in one script.  Should I revert this?

Reply:
Yes please just use one script. Let's try to limit our travis resources as much as possible ;-)